locale/nl/download/package-manager.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: page.hbs
+title: Installeer Node.js via package-manager
+---
+
+# Installeer Node.js via package-manager
+
+***Opmerking:*** De packages op deze pagina worden onderhouden door hun makers, **niet** door het Node.js-kernteam. Meld eventuele problemen die u tegenkomt aan de pakketbeheerders. Als blijkt dat uw probleem een bug is die veroorzaakt wordt door Node.js, dan zullen de pakketbeheerders dit bij ons melden.
+
+---
+
+* [Alpine Linux](#alpine-linux)
+* [Android](#android)
+* [Arch Linux](#arch-linux)
+* [CentOS, Fedora en Red Hat Enterprise Linux](#centos-fedora-and-red-hat-enterprise-linux)
+* [Debian en Ubuntu gebaseerde Linux-distributies](#debian-and-ubuntu-based-linux-distributions)
+* [fnm](#fnm)
+* [FreeBSD](#freebsd)
+* [Gentoo](#gentoo)
+* [IBM i](#ibm-i)
+* [macOS](#macos)
+* [n](#n)
+* [NetBSD](#netbsd)
+* [Nodenv](#nodenv)
+* [nvm](#nvm)
+* [nvs](#nvs)
+* [OpenBSD](#openbsd)
+* [openSUSE en SLE](#opensuse-and-sle)
+* [SmartOS en illumos](#smartos-and-illumos)
+* [Snap](#snap)
+* [Solus](#solus)
+* [Void Linux](#void-linux)
+* [Windows](#windows-1)
+* [z/OS](#zos)
+
+---
+
+## Alpine Linux
+
+Node.js LTS en npm-packages zijn beschikbaar in de Main Repository.
+
+```bash
+apk add nodejs npm
+```
+
+Node.js Current kan geïnstalleerd worden vanuit de Community Repository.
+
+```bash
+apk add nodejs-current
+```
+
+## Android
+
+Android-ondersteuning is nog steeds experimenteel binnen Node.js, dus precompiled-binaries zijn momenteel nog niet beschikbaar.
```

Comment:
```suggestion
Android-ondersteuning is nog steeds experimenteel binnen Node.js, dus precompiled binaries zijn momenteel nog niet beschikbaar.
```
Komt ook op een aantal andere plaatsen voor. Ik zal er één of twee aanwijzen, kijk maar of je er iets mee wil doen. Geen groot struikelblok.